app/views/pages/home.html.erb

```diff
@@ -1,9 +1,30 @@
-<h1 style="color: white;Background-color:black;text-align:center; font-family: Arial, sans-serif">Scaffolding for Address, Location and Admin </h1>
-<h3 style="text-align:center;color:blue" >Click on the below Links to Create, Delete, Update and View</h3>
-<ul style="text-align: center;">
-  <li><%= link_to 'Admin', admins_path, style: "background-color: #993366;color: white;padding: 14px 25px;text-align: center;text-decoration: none;display: inline-block; font-weight: bold;  margin:8px" %></li>
-  <li><%= link_to 'Address', addresses_path, style: "background-color: #336699;color: white;padding: 14px 25px;text-align: center;text-decoration: none;display: inline-block; font-weight: bold; margin:8px" %></li>
-  <li><%= link_to 'Location', locations_path, style: "background-color: #f44336;color: white;padding: 14px 25px;text-align: center;text-decoration: none;display: inline-block; font-weight: bold; margin:8px" %></li>
-
-  </ul>
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title><%= t('login.title')%></title>
+    <%= stylesheet_link_tag "pages_home" %>
+   
+    
+</head>
+<body>
```

Comment:
What is the need to include the hml, head tags ?

Reply:
Yes  sir you are right i am agreed with you that there is no need to include html, head tags. code will work without this tags also. so i have updated the code(removed tags) and pushed on github.